### ISSUE TITLE
where can I download phpquery.php required in your example 2 in readme

### ISSUE THREAD

**Issue by webdlp** [monkeysuffrage/pgbrowser#3]
Hi,
very nice tutorial in readme. I have a simple question, where can I downloaded phpquery.php for your example 2 in readme file? I downloaded one from google but script still did not work. Can you take a look into this script again? Thank you!

**Comment by monkeysuffrage** [comment]
Hi,
Development ended on phpQuery some time ago, I'm currently recommending
this library instead:
http://sourceforge.net/projects/advancedhtmldom/?source=directory

Thanks,
- P

**Comment by bissquitt** [comment]
Can that set fields and submit posts? 
(Side note, are you interested in side projects by chance?)

**Comment by monkeysuffrage** [comment]
Yes, the different parsers are about the `search` and `at` functionality,
submitting forms isn't affected by switching parsers,.
I'm interested in freelance projects and the best way to hire me is to
start a project on elance and invite me to bid on it.

**Comment by webdlp** [comment]
Thank you for advanced html dom.

**Comment by monkeysuffrage** [comment]
You're welcome.
